sdf/1.7/CMakeLists.txt

```diff
@@ -52,26 +55,27 @@ set (sdfs
   state.sdf
   surface.sdf
   transceiver.sdf
+  urdf.sdf
   visual.sdf
   world.sdf
 )
 
 set (SDF_SCHEMA)
 
-foreach(FIL ${sdfs})
-  get_filename_component(ABS_FIL ${FIL} ABSOLUTE)
-  get_filename_component(FIL_WE ${FIL} NAME_WE)
+set(root root.sdf)
```

Comment:
Can you make the same changes to the all the `sdf/*/CMakeLists.txt`?

Reply:
75b33b6d6447182b5945832d8296cf883f55c908 I've made the changes for until 1.5 since 1.4 and older do not print out the schemas